bundles/org.openhab.binding.somfytahoma/src/main/resources/ESH-INF/thing/smokesensor.xml

```diff
@@ -10,6 +10,8 @@
 		<label>Somfy Smoke Sensor</label>
 		<channels>
 			<channel id="contact" typeId="contact"></channel>
+			<channel id="short_check" typeId="short_check"></channel>
```

Comment:
These are not switches as they only send a command. They are better served by combining the short and long channels in a new channel and implement it using the command pattern, see: https://www.openhab.org/docs/developer/bindings/thing-xml.html#command-description

Reply:
changed. I hope I understood it corectly - cannot test it